doc: Bluetooth: Mesh: Clarify AES usage in DFU metadata (#70954)
zephyrproject-rtos/zephyr: doc/connectivity/bluetooth/api/mesh/dfu.rst

|                        |              | with a status message.                 |
+------------------------+--------------+----------------------------------------+

  .. note::

>       In Mesh DFU metadata, AES-CMAC serves as a hashing function with a fixed key, not for

Reviewer: @mia-ko , could you please review?

>   .. note::
> 
>       In Mesh DFU metadata, AES-CMAC serves as a hashing function with a fixed key, not for
>       encryption. The resulting hash isn't secure due to the key being known.

Reviewer 2: ```suggestion
  .. note::

      The AES-CMAC algorithm serves as a hashing function with a fixed key and is not used for
      encryption in Bluetooth Mesh DFU metadata. The resulting hash is not secure since the key is known.
```
